src/app/ticket-not-found/ticket-not-found.component.ts

```diff
@@ -7,9 +9,13 @@ import { Component, OnInit } from '@angular/core';
 })
 export class TicketNotFoundComponent implements OnInit {
 
-  constructor() { }
+  supportedMunicipalities: Municipality[] = [];
+
+  constructor(private municipalityService: MunicipalitiesService) { }
 
   ngOnInit() {
+    this.municipalityService.findSupported(true).subscribe((supportedMunis) => {
```

Comment:
Same thing from the other PR here. If you subscribe, you must unsubscribe. But better than needing to subscribe, store the observable as `supportedMunicipalities$: Observable<Municipality[]>;` and then use the async pipe in the template